### PR TITLE
Ninja teleport kidnap works based on pulling people instead of based on grabs

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**

### CODE REVIEW

**Comment by PKPenguin321** [comment]
bad omens

File: code/modules/ninja/suit/n_suit_verbs/ninja_teleporting.dm [hunk 1]
```diff
@@ -12,12 +12,10 @@ Contents:
...
-	if(istype(H.get_active_hand(),/obj/item/weapon/grab))//Handles grabbed persons.
-		var/obj/item/weapon/grab/G = H.get_active_hand()
-		G.affecting.loc = locate(T.x+rand(-1,1),T.y+rand(-1,1),T.z)//variation of position.
-	if(istype(H.get_inactive_hand(),/obj/item/weapon/grab))
-		var/obj/item/weapon/grab/G = H.get_inactive_hand()
-		G.affecting.loc = locate(T.x+rand(-1,1),T.y+rand(-1,1),T.z)//variation of position.
+	if(H.pulling && (istype(H.pulling, /mob/living)))
+		var/mob/living/victim =	H.pulling
+		if(!victim.anchored)
+			victim.forceMove(locate(T.x+rand(-1,1),T.y+rand(-1,1),T.z))
```
Comment (neersighted) [inline code/modules/ninja/suit/n_suit_verbs/ninja_teleporting.dm]: Space your operators please

**Comment by KorPhaeron** [comment]
@duncathan 

Is there a reason you skipped this one?

**Comment by duncathan** [comment]
I was looking for "removes grab"

**Pull request merged**